src/Tribe/Attendees_Table.php

```diff
@@ -838,28 +838,60 @@ public function prepare_items() {
 			$search_keys = [
 				'purchaser_name',
 				'purchaser_email',
+				'order_status',
 				'product_id',
 				'security_code',
 				'user',
 			];
 
 			/**
-			 * Filters the item keys that should be used to filter attendees while searching them.
+			 * Filters the item keys that can be used to filter attendees while searching them.
 			 *
 			 * @since 4.7
 			 * @since 4.10.6 Deprecated usage of $items attendees list.
 			 *
-			 * @param array  $search_keys The keys that should be used to search attendees.
+			 * @param array  $search_keys The keys that can be used to search attendees.
 			 * @param array  $items       (deprecated) The attendees list.
-			 * @param string $s           The current search string.
+			 * @param string $search      The current search string.
 			 */
 			$search_keys = apply_filters( 'tribe_tickets_search_attendees_by', $search_keys, [], $search );
 
-			// Only get matches that have search phrase in the keys.
-			$args['where_multi'] = [
-				[
-					$search_keys,
-					'LIKE',
+			// Default selection.
+			$search_key = 'purchaser_name';
+
+			if ( ! empty( $_REQUEST['tribe_attendee_search_type'] ) ) {
```

Comment:
Can we make use of `tribe_get_request_var()`.

Reply:
Fixed